password.py

```diff
@@ -1,49 +1,302 @@
 def password_randomizer_query():
+    accepted_answers = 'yes','no'
+
+    while True:
+        rand_answer = input("Do you want a randomized password? ('yes' or 'no'):\n")
+        if rand_answer == 'yes':
+            gen_pass = random_pass()
+            account_info_dict["Password"] = gen_pass
+            print("Your random pass is: " + gen_pass)
+            break
+        elif rand_answer == 'no':
+            break
+        elif rand_answer != accepted_answers:
+            print("Incorrect use.")
+
+    return rand_answer
 
-    return
 # Generates a randomized password using letters and digits
 def random_pass():
+    string = shane_char.ascii_characters_and_num()
+    letters_and_num = string.letters + string.digits
+    rand_password = random.choice(string.lowercase)
+    rand_password += random.choice(string.uppercase)
+    rand_password += random.choice(string.digits)
+    rand_password += random.choice(string.special_characters)
+
+    for characters in range(6):
+        rand_password += random.choice(letters_and_num)
+
+    password_list = list(rand_password)
+    random.SystemRandom().shuffle(password_list)
+    rand_password = ''.join(password_list)
+
+    return rand_password
 
-    return
 # Saves and stores account
 def account_save():
 
-    return
+    account_file = open('account.txt','w')
+    account_file.write(str(account_dict))
+    account_file.close()
+
+    return account_dict
+
+#lists account dict
+# def list_account():
+#     account_file_read = open('account.txt','r')
+#     print(account_file_read.read())
+#     account_file_read.close()
+
 # Recover exisiting account
 def account_recovery():
+    account_dict_file = open('account.txt','r')
+
+    for line in account_dict_file:
+        word = re.split('\{|: |, |\}', line)
+        name = word[1]
+        username = name.replace("'","")
+        pass_w = word[6]
+        password = pass_w.replace("'","")
+        sec_ques = word[8]
+        security_question = sec_ques.replace('"',"")
+        sec_answ = word[10]
+        security_answer = sec_answ.replace("'","")
+    
+    attempts = 4
+
+    while True:
+        print("Enter a username: ")
+        user_answer = input("")
+
+        if user_answer == username:
+            while True:
+                print(security_question)
+                question_answer = input("")
+                if question_answer == security_answer:
+                    print("Your password is: "+password)
+                    break
+                elif attempts == 0:
+                    print("You have exceeded max attempts.")
+                    break
+                else:
+                    attempts -= 1
+                    print("You have "+ str(attempts) +" attempt(s) left.")        
+            break
+        else:
+            print("Username not found!")
+            continue
+
+    account_dict_file.close()
 
     return
 
 #<---- SECOND MAJOR FUNCTION ---->
 # Login to existing account
 def login():
```

Comment:
I'm going to avoid commenting on this function because its fundamentally broken like the above one.